torch/_dynamo/convert_frame.py

```diff
@@ -139,6 +140,36 @@ def _fn(*args, **kwargs):
     return _fn
 
 
+def has_jagged_tensor(frame):
+    """
+    This is a short term solution to skip Dynamo tracing on frame whose inputs
+    are Jagged tensors. These tensors contain a list of integers on the side
+    which in current state causes excessive recompilations and guards.
+
+    We are looking at ways on how to better handle this situation. Meawhile this
+    stop gap solution will allow full model compilation without the slow
+    recompilation.
+    """
+    try:
+        import torchrec  # type: ignore[import]  # noqa:F401
+        from torchrec.sparse.jagged_tensor import (  # type: ignore[import]
+            JaggedTensor,
+            KeyedJaggedTensor,
+        )
```

Comment:
Can we PLEAAASE find some way of matching against external library types that doesn't require importing the modules. This has already caused bugs e.g., https://github.com/Quansight-Labs/numpy_pytorch_interop/issues/166 and causes unexpected behavior (e.g., if the external library has a C extension built against torch, you can cause a spooky segfault at a distance)

Reply:
maybe `obj.__class__.{__name__,__module__}`.